release-notes-4.0.0.md

```diff
@@ -3,6 +3,9 @@
 #### Version Number
 ${version-number}
 
+#### Breaking change
+- US975260: Refactor to remove HPE references in package names
```

Comment:
Is this really a breaking change to the autoscaler?

Reply:
I was not sure if any component references to autoscaler code. Looking at it again, perhaps its not a breaking change.